bash2048.sh

```diff
@@ -262,6 +264,76 @@ function reload_game {
   index_max=board_size-1
 }
 
+# print the content of the highscores file
+function show_highscores {
+    header="Highscore Name Date"
+    printf "%10s | %30s | %30s" "Highscore" "Name" "Date"
+    echo ""
+    echo "----------------------------------------------------------------------------"
+    while IFS=";" read -r score name date
```

Comment:
it seems like this expects highscore file delimited by semicolon character, but it is not saved like that, that results in this:
```
+ IFS=';'
+ read -r score name date
./bash2048.sh: line 273: read: 64 aaa Thu Jul  9 17:30:35 CEST 2020: syntax error in expression (error token is "aaa Thu Jul  9 17:30:35 CEST 2020")
```
for highscore file:
```
64 aaa Thu Jul  9 17:30:35 CEST 2020
40 bbb Thu Jul  9 17:31:00 CEST 2020
```

Reply:
:+1: Fixed